Library/Formula/eg.rb

```diff
@@ -0,0 +1,28 @@
+class Eg < Formula
+  homepage "https://github.com/srsudar/eg"
+  desc "Useful examples for command line tools."
```

Comment:
It’s “command-line”, not “command line” (yes, [we have an audit rule for that](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/26916/version=mavericks/testReport/junit/brew-test-bot/mavericks/audit_eg___strict/)) :wink: